packages/epo-react-lib/src/types/castiron.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module "@castiron/style-mixins" {
```

Comment:
added some minimal typings for the castiron mixins that are used by our style system